Packs/DigitalShadows/Integrations/ReliaQuestGreyMatterDRPIncidents/ReliaQuestGreyMatterDRPIncidents.yml

```
@@ -99,30 +109,36 @@ configuration:
   - very-high
   required: false
   type: 16
+  section: Connect
```

```suggestion
  section: Collect
```
Same as previous comment